src/util/symbol_table_builder.h

```diff
@@ -0,0 +1,113 @@
+// Copyright 2016-2018 Diffblue Limited. All Rights Reserved.
```

Comment:
This is different from any other file headers in use. The "All Rights Reserved." scares me.